### PR TITLE
fix(dashboard): Create workflow drawer fixes

### PR DESCRIPTION
**Pull request by desiprisg**
### What changed? Why was the change needed?

Related enterprise package changes [PR](https://github.com/novuhq/packages-enterprise/pull/236)

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

**Review by SokratisVidros — APPROVED** [review]

File: apps/dashboard/src/utils/slugify.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export function slugify(text: string) {
```
Comment (SokratisVidros) [inline apps/dashboard/src/utils/slugify.ts]: This can co into shared, to be reused between BE and FE

Comment (rifont) [reply apps/dashboard/src/utils/slugify.ts]: @djabarovgeorge , could you please coordinate with @desiprisg to reuse the `slugify` utility you've added in a separate PR?

Comment (SokratisVidros) [reply apps/dashboard/src/utils/slugify.ts]: ☝️

Comment (djabarovgeorge) [reply apps/dashboard/src/utils/slugify.ts]: sure, if we want to use signify function without dependencies so we can share it with the client we need to move this one to Shared package and delete [slugifyName](https://github.com/novuhq/novu/blob/add-env-switch-by-url/libs/application-generic/src/utils/slugify-name.ts#L3) from app-gen so everyone will use this one.

Comment (djabarovgeorge) [reply apps/dashboard/src/utils/slugify.ts]: At the same time, we need to make sure we remove "slugify" from the dependencies across the monorepo so no one will use it.

Comment (LetItRock) [reply apps/dashboard/src/utils/slugify.ts]: I'll update the PR with this suggestion

**Pull request merged**